pkg/build/controller/strategy/docker.go

```diff
@@ -93,6 +97,10 @@ func (bs *DockerBuildStrategy) CreateBuildPod(build *buildv1.Build, includeAddit
 						EmptyDir: &v1.EmptyDirVolumeSource{},
 					},
 				},
+				{Name: "buildcachedir", VolumeSource: v1.VolumeSource{
+					HostPath: &v1.HostPathVolumeSource{Path: "/miloslavcache/"},
```

Comment:
Um… Eventually, `/var/cache/origin/build-metadata` or something like that?

(Either way, does this need SELinux policy updates?)

Reply:
thanks for these comments miloslav, btw, this is just for testing, naming is totally random for now